alicloud/table_alicloud_cs_kubernetes_cluster.go

```diff
@@ -448,6 +456,44 @@ func getCsKubernetesClusterLog(ctx context.Context, d *plugin.QueryData, h *plug
 	return nil, nil
 }
 
+func getCsKubernetesClusterNameSpaces(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
+	region := plugin.GetMatrixItem(ctx)[matrixKeyRegion].(string)
+	plugin.Logger(ctx).Trace("getCsKubernetesClusterNameSpaces")
+
+	var id string
+	if h.Item != nil {
+		clusterData := h.Item.(map[string]interface{})
+		id = clusterData["cluster_id"].(string)
+	} else {
+		id = d.KeyColumnQuals["cluster_id"].GetStringValue()
+	}
+
+	accessKey := os.Getenv("ALICLOUD_ACCESS_KEY")
+	secretAccess := os.Getenv("ALICLOUD_SECRET_KEY")
+	client, err := sdk.NewClientWithAccessKey(region, accessKey, secretAccess)
```

Comment:
we could not use ContainerService to fetch the client ?

Reply:
ContainerService client has been used